sql/query/update_test.go

```diff
@@ -36,13 +47,13 @@ func TestUpdateStmt(t *testing.T) {
 			require.NoError(t, err)
 			defer db.Close()
 
-			err = db.Exec("CREATE TABLE test")
+			err = db.Exec("CREATE TABLE test (a text not null)")
 			require.NoError(t, err)
 			err = db.Exec("INSERT INTO test (a, b, c) VALUES ('foo1', 'bar1', 'baz1')")
 			require.NoError(t, err)
 			err = db.Exec("INSERT INTO test (a, b) VALUES ('foo2', 'bar2')")
 			require.NoError(t, err)
-			err = db.Exec("INSERT INTO test (d, e) VALUES ('foo3', 'bar3')")
+			err = db.Exec("INSERT INTO test (a, d, e) VALUES ('foo3', 'bar3', 'baz3')")
```

Comment:
The fact that `a` was missing in this document was to make sure we don't get an error if we try to SET a field that doesn't exist. I think we should keep this kind of case to prevent regressions, wdyt?

Reply:
I think that we still have this test when we set `f` which doesn't exist in the document:
> {"SET / No cond / with missing field", "UPDATE test SET f = 'boo'", false, `[{"a":"foo1","b":"bar1","c":"baz1","f":"boo"},{"a":"foo2","b":"bar2","f":"boo"},{"a":"foo3","d":"bar3","e":"baz3","f":"boo"}]`, nil}

Does it make sense?

Reply:
Yes, but not with the `WHERE` clause: `WHERE a = ?`, `WHERE a = f`

Reply:
There is a test with the `WHERE` clause actually:
> {"SET / With cond / with missing field", "UPDATE test SET f = 'boo' WHERE d = 'bar3'", false, `[{"a":"foo1","b":"bar1","c":"baz1"},{"a":"foo2","b":"bar2"},{"a":"foo3","d":"bar3","e":"baz3","f":"boo"}]`, nil},

Reply:
Indeed 👍